lib/fedora_migrate/datastream_verification.rb

```diff
@@ -0,0 +1,43 @@
+module FedoraMigrate::DatastreamVerification
+  
+  attr_accessor :datastream
+
+  def valid? datastream=nil
+    @datastream = datastream || source
+    return true if no_checksum?
+    return true if checksums_match?
+    return true if content_equivalent? 
+    FedoraMigrate::Logger.warn "#{source.pid} datastream #{datastream.dsid} validation failed"
+    return false
+  end
+
+  def checksums_match?
+    datastream.checksum == target_checksum.split(/:/).last
```

Comment:
Do you think the `.split(/:/.last` should be in the matching method or in `target_checksum`?